tests/pytests/functional/modules/test_chocolatey.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from salt.exceptions import CommandExecutionError
+
+pytestmark = [
+    pytest.mark.destructive,
```

Comment:
```suggestion
    pytest.mark.destructive_test,
```
There is no `pytest.mark.destructive` mark

Reply:
Done